### DISCUSSION TITLE
disable video in tailwindcss - does not work with Augmented reality

### DISCUSSION THREAD

**Discussion by kordou** [tailwindlabs/tailwindcss#9747] [Help]
Hi all,
I am trying yo use tailwind in an AR webApp. 
The isssue that all ar Apps use the Video in order to display the camera and tailwind creates the following

img video{
  max-width: 100%;
  height: auto;
}

img,
svg,
canvas,
audio,
iframe,
embed,
video
object {
  display: block;
  /* 1 */
  vertical-align: middle;
  /* 2 */
}

that dows not allow for a fullscreen video camera.

I tryed to a @base video with correct sizes but nothing.

the only way that works it to remove the   video from the two above.

I am looking for a way (Not manual deletion) in order to able to be user with parcel (bandler) 

any help here ??

thanks

**Comment by wongjn** [comment]
You could look at [disabling preflight](https://tailwindcss.com/docs/preflight#disabling-preflight) which is where these styles come from, but this may impact the styling in other areas too.

**Reply by kordou** [reply]
wongjn Thank you , but if i am not corect wiil disable all the corePlugins correct ? not only the video element.

i need a way to remove only the video

**Reply by wongjn** [reply]
Indeed. You could disable the core preflight plugin and then create your own plugin that adapts [the core implementation](https://github.com/tailwindlabs/tailwindcss/blob/30d25950836ff1a3f3bc90face7bca4c7ebeb73c/src/corePlugins.js#L463-L473) but removing the video element selector from any rules.

**Reply by kordou** [reply]
You gave a great Idea and a solution !!!

remove the video (for that project) from the original prefil file in  node_modules/tailwindcss/lib/css/preflight.css!

Works fine!! 

I think that Tailwind shoud think about that issue because augmented and Virtual Reality apps will grow very fast and it could be a grate lib for for webXR
